rust/schema.py

```diff
@@ -119,15 +118,31 @@ class Function(Declaration):
     body: Expr | child
 
 
-#     Missing,
+@rust.doc_test_signature("() -> ()")
 class MissingExpr(Expr):
+    """
+    A missing expression, used as a place holder for incomplete syntax, as well as bodies of functions that are defined externally.
+
+    ```
+    let x = non_existing_macro!();
+    ```
+    """
     pass
 
 
 #     Path(Path),
 
-
+@rust.doc_test_signature("() -> ()")
 class PathExpr(Expr):
+    """
+    A path expression. For example:
+    ```
+    let x = variable;
+    let x = foo::bar;
+    let y = <T>::foo
+    let z = <Type as Trait>::foo
```

Comment:
hmm, how does rust-analyzer and our extractor currently fare with undefined symbols like the ones here? Aren't we getting missing things rather than what we want to test? Also, there might be some semicolons missing.

Reply:
The extractor and rust-analyzer are fine with missing symbols. Things like jump to definition won't work in the IDE, but the code is still recognized as syntactically valid.

Yeah, I should have added those semicolons, but the parser is quite resilient, so I hadn't noticed. I think it would be nice to also generate a sort of coverage test query that checks that all things appear at least once in the examples.